packages/backend/src/nest/connections-manager/connections-manager.service.ts

```diff
@@ -609,8 +724,8 @@ export class ConnectionsManagerService extends EventEmitter implements OnModuleI
     this.storageService.on(StorageEvents.MESSAGE_MEDIA_UPDATED, (payload: FileMetadata) => {
       this.serverIoProvider.io.emit(SocketActionTypes.MESSAGE_MEDIA_UPDATED, payload)
     })
-    this.storageService.on(StorageEvents.UPDATE_PEERS_LIST, (payload: StorePeerListPayload) => {
-      this.serverIoProvider.io.emit(SocketActionTypes.PEER_LIST, payload)
```

Comment:
Instead of sending an event for a single piece of data (e.g. peerList), we send a general event (COMMUNITY_UPDATED) anytime the backend updates the Community model so that the frontend can replicate the state.